quiz.py

```diff
@@ -0,0 +1,63 @@
+def startQuiz():
+    while (True):
+        print("Start Quiz")
+        num_questions = howManyQuestions("samplequiz.txt") #count number of quesitons in the file.
+        displayQuestion("samplequiz.txt") #display the question.
+        #would be a list that contains user answers.
+        answers = [input("Your answer for question {}:\n".format(i+1)) for i in range(num_questions)] 
+        num_correct = checkAnswers("samplequiz.txt", answers) #check num of correct answers
+        final_score = calculateScore(num_correct, num_questions) #calculate final score
+        print("Your Final Score is: {}%".format(final_score)) 
+        #shows a comparison between your answer and the correct answer 
+        compareAnswers("samplequiz.txt", answers)
+        again = input("Restart? Y/N\n")
+        if (again == "N" or again == "n"):
+            break
+
+#Function that takes in a quiz file and returns a list filled with the correct answers sequentially listed
+def getCorrectAnswers(filename):
+    fileObject = open(filename, 'r')
+    num_questions = int(fileObject.readline())
+    answer_lines = fileObject.readlines()[num_questions*2+1:]
+    for i in range(len(answer_lines)):
+        answer_lines[i] = answer_lines[i].strip('\n').strip()
+    fileObject.close()
+    return answer_lines
+
+#Function to print a comparison between your answer and the correct answer
+def compareAnswers(filename, answers):
+    correct_answer = getCorrectAnswers(filename)
+    print("Your Answer \t Correct Answer\n")
+    for i in range(len(correct_answer)):
+        print ("{} \t\t {}\n".format(answers[i], correct_answer[i]))
+    
+#Function to get the number of question in the quiz file.
+def howManyQuestions(filename):
+    fileObject = open(filename, 'r')
+    num_lines = int(fileObject.readline())
+    fileObject.close()
+    return num_lines
+
+#Function that displays the question in the quiz file
+def displayQuestion(filename):
+    fileObject = open(filename, 'r')
+    num_questions = int(fileObject.readline())
+    for i in range(num_questions*2):
+        print(fileObject.readline())
+    fileObject.close()
+
+#Function to check the answers and returns the number of correct answer
+def checkAnswers(filename, answer_list):
```

Comment:
Since the answers are the multiple choice letters (A,B,C,D) and the user's answers are numeric, the function does not correctly calculate the score